docs/Getting Started/Connecting to an Infotainment System/index.md

```diff
@@ -99,24 +109,52 @@ If you are using Sdl Core and Generic HMI, you will have to add a policy table e
  }
 ```
 
-Fore more information about policy tables please visit [Plicy Tables Guides](https://smartdevicelink.com/en/guides/sdl-server/api-reference-documentation/policy-table/overview).
+For more information about policy tables please visit the [Policy Table](https://smartdevicelink.com/en/guides/sdl-server/api-reference-documentation/policy-table/overview) guide.
+
+### Manticore
+If you are using Manticore, the app connection information can be easily added in the settings tab of the Manticore web page. Please note that Manticore needs to access your machine's IP address in order to be able to start a websocket connection with your app. If you are hosting the app on your local machine, you may need to do extra setup to make your machine publicly accessible.
+
+![Manticore - Cloud and Embedded App Settings](assets/Manticore_cloud_embedded_app_settings.png)
+
+## Running the SDL App
+Once you have a configured instance of Core running, you should see your SDL app name and icon appear on HMI. However, nothing will happen when you tap on your SDL app icon until you build and run your SDL app.
+
+![Generic - SDL Apps Tab](assets/Generic_apps_screen.png)
+
+Once your SDL app is running, either locally in an IDE or on a server, you will be able to launch the SDL app by clicking on the app icon in the HMI.
+
+![Generic - SDL App Main Screen](assets/Generic_non_media.png)
+
+This is the main screen of your SDL app. If you get to this point, your SDL app is working.
+!@
+
+## How Do I Find Out What Features My Infotainment System Supports?
```

Comment:
I don't think that putting this section here is helpful. It's at the bottom of a somewhat unrelated guide and if you didn't write this and I asked you to find it in the guides, I doubt this is where you'd first look. Please move it to "Integration Basics" or a separate guide. I probably slightly prefer a separate guide.

Reply:
Fixed. Moved to a new section